utils/logalloc.cc

```diff
@@ -2034,17 +2034,17 @@ std::ostream& operator<<(std::ostream& out, const occupancy_stats& stats) {
         stats.used_fraction() * 100, stats.used_space(), stats.total_space());
 }
 
-occupancy_stats tracker::impl::region_occupancy() noexcept {
-    reclaiming_lock _(*this);
+occupancy_stats tracker::impl::region_occupancy() const noexcept {
+    reclaiming_lock _(*const_cast<tracker::impl*>(this));
```

Comment:
It does.  It touches `_reclaiming_enabled`.
Coming to think about it, `tracker::impl::region_occupancy` and `tracker::impl::occupancy` are synchronous.
@tgrabiec you added this in d20fae96a2da5d0fe381c91c42be6253503eb795
Why do we need the `reclaiming_lock` at all in those stats functions?

Reply:
Reclaiming invalidates iterators into _regions, so it's a safety measure. In case of occupancy(), reclaiming should not happen because it doesn't currently allocate memory. I guess it was added there just in case.

Reply:
`reclaiming_lock` deleted in these functions in v9